### PR TITLE
added translation on blog page #6389

### PR DESCRIPTION
**Pull request by gautampanday**
Implemented translation tag at blog page

### CODE REVIEW

File: theme/templates/blogs.html [hunk 1]
```diff
@@ -38,16 +38,16 @@ <h1 class="white-text-color margin-0">Blog</h1>
              </td>
            </tr>
          </table>
-         <h3 class="margin-0"><a href="{{ SITEURL }}/{{ article.url }}">{{ article.title }}</a></h3>
+         <h3 class="margin-0"><a href="{{ SITEURL }}/{{ article.url }}">{% trans %}{{ article.title }}{% endtrans %}</a></h3>
```
Comment (tarunbhardwaj) [inline theme/templates/blogs.html]: variable cannot have trans, so no need

File: theme/templates/blogs.html [hunk 2]
```diff
@@ -38,16 +38,16 @@ <h1 class="white-text-color margin-0">Blog</h1>
...
          <ul class="mar_bottom3">
-          <li class="pull-left list-unstyled post-by">{{article.author}}</li>
+          <li class="pull-left list-unstyled post-by">{% trans %}{{article.author}}{% endtrans %}</li>
```
Comment (tarunbhardwaj) [inline theme/templates/blogs.html]: no need

File: theme/templates/blogs.html [hunk 3]
```diff
@@ -38,16 +38,16 @@ <h1 class="white-text-color margin-0">Blog</h1>
...
           <li class="list-unstyled post-event pull-left">
-            <a class="post-category" href="{{ SITEURL }}/{{ article.category.url }}">{{ article.category }}</a>
+            <a class="post-category" href="{{ SITEURL }}/{{ article.category.url }}">{% trans %}{{ article.category }}{% endtrans %}</a>
```
Comment (tarunbhardwaj) [inline theme/templates/blogs.html]: no need

File: theme/templates/blogs.html [hunk 4]
```diff
@@ -38,16 +38,16 @@ <h1 class="white-text-color margin-0">Blog</h1>
...
           </li>
         </ul>
       </div>
       <div class="clearfix"></div>
-      {{ article.summary }}
+      {% trans %}{{ article.summary }}{% endtrans %}
```
Comment (tarunbhardwaj) [inline theme/templates/blogs.html]: no need

**Comment by sharoonthomas** [comment]
:+1:

**Pull request merged**